app/jobs/fetch_all_active_ama_appeals_job.rb

```diff
@@ -80,8 +80,15 @@ def map_appeal_privacy_act_state(appeal)
     { privacy_act_pending: false, privacy_act_complete: false }
   end
 
+  # Purpose: Determines if the hearing scheduled attribute within the associated
+  # appeal_state record should be set to true
+  # Params: Appeal object
+  # Returns: Hash of "hearing scheduled" key value pair
   def map_appeal_hearing_scheduled_state(appeal)
-    # Code goes here ...
+    if appeal.hearings.count > 0 && appeal.hearings.max_by(&:id).disposition.nil?
+      return { hearing_scheduled: true }
+    end
+
```

Comment:
```suggestion
    if appeal&.hearings.count > 0 && appeal.hearings.max_by(&:id).disposition.nil?
      return { hearing_scheduled: true }
    end

```

I think we shoudl protect against a nil being passed into the function. If someone passes a nil it will me a no def method error and derail processing